(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'l' is not defined"
p3
sS'layer'
p4
S'/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f23b5158330, file "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 9>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/leonardo/Downloads/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 226, in <module>\n  File "/home/leonardo/Downloads/web2py/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/leonardo/Downloads/web2py/gluon/tools.py", line 3982, in f\n    return action(*a, **b)\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 186, in busca_aluno\n    if db((db.frequencia.matric_aluno == l.id) & (db.frequencia.cod_presenca == cod_presenca)).select():\nNameError: global name \'l\' is not defined\n'
p11
s.